cloud_functions/spritesheet_tests.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import mock
+from google.cloud import storage, datastore
+import create_spritesheet
+
+NAME = 'bosticc@google.com/coolest_man_alive/original_images/myles_hun.jpg'
+PREFIX = 'bosticc@google.com/coolest_man_alive/original_images/'
+
+# todo... add more tests.. wanted to get a pr in.
+
+# This functions tests for the correct metadata retrieval of the file
+def test_metadata_retrieval(capsys):
+    key = 'image-count:',
+    number = '1'
+
+    context = mock.MagicMock()
+    context.event_id = 'step-2020-johndallard'
+    context.event_type = 'gcs-event'
+
+    # Call tested function
+    create_spritesheet.get_metadata('bosticc@google.com', 'coolest_man_alive')
+    out, err = capsys.readouterr()
```

Comment:
I don't understand this test (and all the ones below).
Why are you not checking the return value of get_metadata? Your test should NOT rely on the log message

Reply:
1) Changing the tests and comments now
2) Sorry, those log statements were from debugging.